src/landIce/evaluators/LandIce_EnthalpyBasalResid_Def.hpp

```diff
@@ -81,9 +81,8 @@ operator() (const Enthalpy_Basal_Residual_Tag& tag, const int& sideSet_idx) cons
   const int cell = sideSet.ws_elem_idx.d_view(sideSet_idx);
   const int side = sideSet.side_pos.d_view(sideSet_idx);
 
-  ScalarT val[maxNumNodesPerSide];
+  ScalarT val[maxNumNodesPerSide] = {};
   for (unsigned int node = 0; node < numSideNodes; ++node) {
-      val[node] = 0;
```

Comment:
Are we 100% sure that the default ctor of a sacado type is to init it as if with ScalarT(0)? If so, then this is fine.

Reply:
Looking at the docs for DFad and SimpleFad, it looks like the default constructors do indeed initialize to 0.